examples/image-streams/image-streams-centos7.json

```diff
@@ -690,7 +690,7 @@
               "openshift.io/display-name": "MongoDB 2.4",
               "description": "Provides a MongoDB 2.4 database on CentOS 7. For more information about using this database image, including OpenShift considerations, see https://github.com/sclorg/mongodb-container/tree/master/2.4/README.md.",
               "iconClass": "icon-mongodb",
-              "tags": "mongodb",
+              "tags": "hidden,mongodb",
```

Comment:
@bparees Non-petset replica set template uses this version. Do we have plans on it? Replace it completely or update to use 2.6 instead?